py2store/persisters/sqlserver_w_pyodbc.py

```diff
@@ -0,0 +1,121 @@
+import platform
+import subprocess
+from collections.abc import MutableMapping
+
+
+class SQLServerPersister(MutableMapping):
+    def __init__(self, conn_protocol='tcp', host='localhost', port='1433', db_username='SA', db_pass='Admin123x',
+                 db_name='py2store', table_name='person', primary_key='id', data_fields=('name',)):
+
+        self.__check_dependencies()
+        self._sql_server_client = pyodbc.connect('DRIVER={{ODBC Driver 17 for SQL Server}};'
+                                                 'SERVER={}:{},{};'
+                                                 'DATABASE={};'
+                                                 'UID={};'
+                                                 'PWD={}'
+                                                 .format(conn_protocol, host, port, db_name, db_username, db_pass))
+
+        self._cursor = self._sql_server_client.cursor()
+        self._table_name = table_name
+        self._primary_key = primary_key
+
+        self._select_all_query = "SELECT * from {table};".format(table=self._table_name)
+        self._insert_query = "INSERT into {table}({{attributes}}) VALUES ({{values}});".format(table=self._table_name)
+        self._select_query = "SELECT * from {table} where {primary_key}='{{value}}'".format(
+            table=self._table_name, primary_key=self._primary_key)
+        self._del_query = "DELETE from {table} where {primary_key} = {{value}};".format(table=self._table_name,
+                                                                                        primary_key=self._primary_key)
+
+    @staticmethod
+    def __check_dependencies():
```

Comment:
Please look how it's implemented in the other persisters and adjust your code accordingly:
https://github.com/i2mint/py2store/blob/master/py2store/persisters/sql_w_sqlalchemy.py#L3